{"uid":"18756df99b90e4b2","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"9bbc2729137fa409b9bc8067987e1d0d","time":{"start":1709946345272,"stop":1709946355539,"duration":10267},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563724bebe93 <unknown>\n#1 0x5637248e3ce6 <unknown>\n#2 0x56372492ee48 <unknown>\n#3 0x56372492ef01 <unknown>\n#4 0x5637249723f4 <unknown>\n#5 0x563724950edd <unknown>\n#6 0x56372496f899 <unknown>\n#7 0x563724950c53 <unknown>\n#8 0x563724921db3 <unknown>\n#9 0x56372492277e <unknown>\n#10 0x563724bb17cb <unknown>\n#11 0x563724bb57e5 <unknown>\n#12 0x563724b9f0e1 <unknown>\n#13 0x563724bb6372 <unknown>\n#14 0x563724b831bf <unknown>\n#15 0x563724bda488 <unknown>\n#16 0x563724bda683 <unknown>\n#17 0x563724beb044 <unknown>\n#18 0x7f7207294ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f55e2edfd10>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...ct='Non Existing', page_title='My Workspace', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"960dc2746e9b54dc3ce3ac274e983f5b\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f55e25454e0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x563724bebe93 <unknown>\nE       #1 0x5637248e3ce6 <unknown>\nE       #2 0x56372492ee48 <unknown>\nE       #3 0x56372492ef01 <unknown>\nE       #4 0x5637249723f4 <unknown>\nE       #5 0x563724950edd <unknown>\nE       #6 0x56372496f899 <unknown>\nE       #7 0x563724950c53 <unknown>\nE       #8 0x563724921db3 <unknown>\nE       #9 0x56372492277e <unknown>\nE       #10 0x563724bb17cb <unknown>\nE       #11 0x563724bb57e5 <unknown>\nE       #12 0x563724b9f0e1 <unknown>\nE       #13 0x563724bb6372 <unknown>\nE       #14 0x563724b831bf <unknown>\nE       #15 0x563724bda488 <unknown>\nE       #16 0x563724bda683 <unknown>\nE       #17 0x563724beb044 <unknown>\nE       #18 0x7f7207294ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"data","time":{"start":1709946297430,"stop":1709946297431,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1709946297430,"stop":1709946297430,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1709946297430,"stop":1709946297430,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563724bebe93 <unknown>\n#1 0x5637248e3ce6 <unknown>\n#2 0x56372492ee48 <unknown>\n#3 0x56372492ef01 <unknown>\n#4 0x5637249723f4 <unknown>\n#5 0x563724950edd <unknown>\n#6 0x56372496f899 <unknown>\n#7 0x563724950c53 <unknown>\n#8 0x563724921db3 <unknown>\n#9 0x56372492277e <unknown>\n#10 0x563724bb17cb <unknown>\n#11 0x563724bb57e5 <unknown>\n#12 0x563724b9f0e1 <unknown>\n#13 0x563724bb6372 <unknown>\n#14 0x563724b831bf <unknown>\n#15 0x563724bda488 <unknown>\n#16 0x563724bda683 <unknown>\n#17 0x563724beb044 <unknown>\n#18 0x7f7207294ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f55e2edfd10>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...ct='Non Existing', page_title='My Workspace', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"960dc2746e9b54dc3ce3ac274e983f5b\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f55e25454e0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x563724bebe93 <unknown>\nE       #1 0x5637248e3ce6 <unknown>\nE       #2 0x56372492ee48 <unknown>\nE       #3 0x56372492ef01 <unknown>\nE       #4 0x5637249723f4 <unknown>\nE       #5 0x563724950edd <unknown>\nE       #6 0x56372496f899 <unknown>\nE       #7 0x563724950c53 <unknown>\nE       #8 0x563724921db3 <unknown>\nE       #9 0x56372492277e <unknown>\nE       #10 0x563724bb17cb <unknown>\nE       #11 0x563724bb57e5 <unknown>\nE       #12 0x563724b9f0e1 <unknown>\nE       #13 0x563724bb6372 <unknown>\nE       #14 0x563724b831bf <unknown>\nE       #15 0x563724bda488 <unknown>\nE       #16 0x563724bda683 <unknown>\nE       #17 0x563724beb044 <unknown>\nE       #18 0x7f7207294ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1709946345272,"stop":1709946355539,"duration":10267},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x563724bebe93 <unknown>\n#1 0x5637248e3ce6 <unknown>\n#2 0x56372492ee48 <unknown>\n#3 0x56372492ef01 <unknown>\n#4 0x5637249723f4 <unknown>\n#5 0x563724950edd <unknown>\n#6 0x56372496f899 <unknown>\n#7 0x563724950c53 <unknown>\n#8 0x563724921db3 <unknown>\n#9 0x56372492277e <unknown>\n#10 0x563724bb17cb <unknown>\n#11 0x563724bb57e5 <unknown>\n#12 0x563724b9f0e1 <unknown>\n#13 0x563724bb6372 <unknown>\n#14 0x563724b831bf <unknown>\n#15 0x563724bda488 <unknown>\n#16 0x563724bda683 <unknown>\n#17 0x563724beb044 <unknown>\n#18 0x7f7207294ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"b01dbc8fea124179","name":"public ip address","source":"b01dbc8fea124179.txt","type":"text/plain","size":13},{"uid":"b9215c278127a5b6","name":"Cookies","source":"b9215c278127a5b6.json","type":"application/json","size":3100},{"uid":"f25370a22a10b040","name":"Session Storage","source":"f25370a22a10b040.json","type":"application/json","size":359},{"uid":"deefb0e64009eda5","name":"Local Storage","source":"deefb0e64009eda5.json","type":"application/json","size":208},{"uid":"bc4080c2a068ba3d","name":"Console Logs","source":"bc4080c2a068ba3d.json","type":"application/json","size":894},{"uid":"35105b971f8ee261","name":"Full Page Screenshot","source":"35105b971f8ee261.png","type":"image/png","size":6186843},{"uid":"7fa0b104a183a72d","name":"URL","source":"7fa0b104a183a72d.uri","type":"text/uri-list","size":23},{"uid":"ee4618c1f2ccaaae","name":"Network Logs","source":"ee4618c1f2ccaaae.json","type":"application/json","size":23259}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"blocker"},{"name":"epic","value":"Security"},{"name":"feature","value":"Login"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az842-374"},{"name":"thread","value":"2428-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'elias@gmail.com'"},{"name":"password","value":"'12345Tr'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"18756df99b90e4b2.json","parameterValues":["'elias@gmail.com'","'12345Tr'"]}